cns/networkcontainers/networkcontainers_windows.go

```diff
@@ -167,7 +176,10 @@ func deleteInterface(networkContainerID string) error {
 
 	log.Printf("[Azure CNS] Going to delete network loopback adapter: %v", args)
 	c := exec.Command("cmd", args...)
```

Comment:
Shouldn't we lock before this line?

Reply:
https://github.com/Azure/azure-container-networking/pull/311/files